A6lib.cpp

```diff
@@ -25,7 +25,8 @@
 //
 
 A6lib::A6lib(int transmitPin, int receivePin) {
-    A6conn = new SoftwareSerial(receivePin, transmitPin, false, 1024);
+    A6conn = new SoftwareSerial(receivePin, transmitPin, false);
```

Comment:
I think a lot of code depends on the 1024 here, so the responses aren't split into multiple pieces. Why was it removed (I'm assuming some version of `SoftSerial` is not compatible with it?) and what's the default in that case?